scipy/sparse/data.py

```diff
@@ -136,7 +136,7 @@ def method(self):
             return self._with_data(result, copy=True)
 
         method.__doc__ = ("Element-wise %s.\n\n"
-                          "See numpy.%s for more information." % (name, name))
+                          "See NumPy.%s for more information." % (name, name))
```

Comment:
this is a function name, so should be lowercase. perhaps add backticks for clarity?

Reply:
Of course.. I was on autopilot every time I noticed 'numpy' :-)